realtime/utils.js

```diff
@@ -376,7 +378,9 @@ function isIpWhitelisted(addr, whitelist) {
  * @returns {Promise} - which resolves to a complete sample with its subject and
  *   aspect.
  */
-function attachAspectSubject(sample, subjectModel, aspectModel) {
+function attachAspectSubject(sample) {
+  const sequelize = require('../db').sequelize;
```

Comment:
If this is necessary to require *here* inside the fn (as opposed to doing the require at the module level), please add a comment explaining why.